<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#page
{background-image: url(:/img/1.jpeg);}

#page_2
{	
background-image: url(:/img/12.jpeg);}

#page_3
{	
background-image: url(:/img/12.jpeg);}
#page_4
{	
background-image: url(:/img/12.jpeg);}
#page_5
{background-image: url(:/img/12.jpeg);}


#page_6
{	
background-image: url(:/img/12.jpeg);}

#page_7
{	
background-image: url(:/img/12.jpeg);}
#page_8
{	
background-image: url(:/img/12.jpeg);}</string>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_inregistrare">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>135</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_inregistrare{
	font: 12pt &quot;Times New Roman&quot;;
		background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inregistrare:hover{
	font: 12pt &quot;Times New Roman&quot;;

background-color:  rgb(179, 255, 0);
border-radius:5px;

}</string>
      </property>
      <property name="text">
       <string>Autentificare</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_autentificare">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>135</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_autentificare{
	font: 12pt &quot;Times New Roman&quot;;
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_autentificare:hover{
	font: 12pt &quot;Times New Roman&quot;;
	
background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
      </property>
      <property name="text">
       <string>Înregistrare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="title">
       <string>Conectare</string>
      </property>
      <widget class="QLabel" name="label_nume_utilizator">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Nume utilizator</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_parila">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Parolă</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nume_utilizator">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_parola">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_conectare">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>250</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_conectare{
	font: 12pt &quot;Times New Roman&quot;;
		background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_conectare:hover{
	font: 12pt &quot;Times New Roman&quot;;

background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Conectare</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inapoi">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi{
	font: 12pt &quot;Times New Roman&quot;;
		background-color: rgb(0, 170, 0);
	
 border-radius:8px;
}
#pushButton_inapoi:hover{
	font: 12pt &quot;Times New Roman&quot;;

background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="title">
       <string>Autentificare</string>
      </property>
      <widget class="QLabel" name="label_nume">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>37</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Nume</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenume">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>37</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenume</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_adresa_de_mail">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresa de mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_varsta">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>37</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Vârstă</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_inaltime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Înălțime</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_greutate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Greutate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nume">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenume">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adres_de_mail">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_varsta">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_inaltime">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_greutate">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>180</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inregistrare_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>290</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inregistrare_2{
	font: 12pt &quot;Times New Roman&quot;;
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inregistrare_2:hover{
	font: 12pt &quot;Times New Roman&quot;;

	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înregistrare</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nume_utilizato">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nume utilizator</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_parola">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>37</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Parolă</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nume_utiliazator">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>220</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_parola_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inapoi_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_2{
	font: 12pt &quot;Times New Roman&quot;;
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inapoi_2:hover{
	font: 12pt &quot;Times New Roman&quot;;
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="title">
       <string>Meniu</string>
      </property>
      <widget class="QPushButton" name="pushButton_actualizare_date">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_actualizare_date{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_actualizare_date:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Actualizare date</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_vizualizare_evolutie">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_vizualizare_evolutie{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_vizualizare_evolutie:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Vizualizare evoluție</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_antrenamente">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_antrenamente{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_antrenamente:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Antrenamente</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_prieteni">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_prieteni{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_prieteni:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Prieteni</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inapoi_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_3{
	
	background-color: rgb(0, 170, 0);
	
 border-radius:8px;
}
#pushButton_inapoi_3:hover{
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="title">
       <string>Prieteni</string>
      </property>
      <widget class="QPushButton" name="pushButton_cereri">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_cereri{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_cereri:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Cereri de prietenie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inapoi_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_4{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inapoi_4:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="title">
       <string>Actualizare date</string>
      </property>
      <widget class="QLabel" name="label_grutate">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Greutate</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_data">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_data">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_greutate_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_salveaza">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>260</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_salveaza{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_salveaza:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Salvează</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inapoi_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_5{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inapoi_5:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-image: url(:/img/1.png);</string>
      </property>
      <property name="title">
       <string>Antrenamente</string>
      </property>
      <widget class="QPushButton" name="pushButton_inapoi_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_6{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inapoi_6:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="title">
       <string>Evolutie</string>
      </property>
      <widget class="QPushButton" name="pushButton_inapoi_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_inapoi_7{
	
	background-color: rgb(0, 170, 0);
 border-radius:8px;
}
#pushButton_inapoi_7:hover{
	
	background-color: rgb(179, 255, 0);
border-radius:5px;

}</string>
       </property>
       <property name="text">
        <string>Înapoi</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
